terra/utils.py

```diff
@@ -20,13 +20,24 @@ def current_fiscal_year_object(today=None):
     return FY.FiscalYear(year)
 
 
+def current_fiscal_year_int(today=None):
+    year = current_fiscal_year(today=today)
```

Comment:
This line looks like never ending recusion

Reply:
This is used in the unit list view to get the year integer rather than FY20xx so the default for unit reports is the current fiscal year.